_data/simple-icons.json

```
@@ -9487,6 +9487,12 @@
             "source": "https://opnsense.org/about/legal-notices/",
             "guidelines": "https://opnsense.org/about/legal-notices/"
         },
+        {
+            "title": "Oppo",
```

```suggestion
            "title": "OPPO",
```

See this: https://www.oppo.com/en/about/.

They call themself `OPPO`.

---

Fixed